lib/src/widgets/pressable/pressable_widget.dart

```diff
@@ -157,35 +160,48 @@ class PressableWidgetState extends State<Pressable> {
         button: true,
         focusable: onEnabled && _node.canRequestFocus,
         focused: _node.hasFocus,
-        child: GestureDetector(
```

Comment:
I think the disabled should behave only for the onTap behavior, we also should not modify the element in the tree through the function returning a widget.

Reply:
I'll change the function. However, about the disabled state only affects `onTap`. Are you sure? In my opinion when disabled is `true`, all actions involving the `Pressable` should be deactivated.

Reply:
@tilucasoli makes sense!